README.md

```diff
@@ -357,6 +357,10 @@ yarn build:ts -w
 Tests for each package reside in `packages/src/*/tests`, and are run using
 `jest`.
 
+### Simulating a Publish with Verdaccio
+We can easily simulate a publish using [Verdaccio](https://verdaccio.org/) which is a private npm proxy registry. For installation check [this link](https://verdaccio.org/docs/installation).
+
+Once it has been installed and executed, you can run `yarn release:local` to simulate a publish.
```

Comment:
```suggestion
Once it has been installed and executed, you can run `yarn release:local` to simulate a publish.

```

Reply:
Prettier ignores markdown since https://github.com/Zilliqa/Zilliqa-JavaScript-Library/commit/3c3d9bd42c12a91aaa96cac48a7a38513d099ea2. I will let prettier format markdown files.